advanced/list-of-log-names.md

```diff
@@ -1,72 +1,76 @@
 ---
-description: In this page all the available logs are listed
+description: On this page all the available logs are listed
 ---
 
 # List of log names
 
-## Categories
+Logs, categorized by what they do are listed here, and are **case sensitive**. Individual logs are written in _UPPER\_CASE_, while category names(which can be used in place of individual log names, see [configuring channels](../admin-commands/config/configuring-channels.md)) are _Capitalized_. Each one of them is separated by categories in the lists below. If you want to log everything, use an asterisk to include all things that can be logged, including ones that will be introduced in future updates.
```

Comment:
1. No space names(which
2. "configuring channels" page link here is incorrect and points to the old format
3. Too little emphasis on the config link anyways, think of someone landing for the first time on this page and thinking "okay the list, what do I do with this?"